Wikipedia/Code/PageEditorViewController.swift

```diff
@@ -248,7 +248,10 @@ final class PageEditorViewController: UIViewController {
             } else {
                 self.addChildEditor(wikitext: wikitextFetchResponse.wikitext, needsReadOnly: needsReadOnly, onloadSelectRange: wikitextFetchResponse.onloadSelectRange)
             }
-            
+            if !UserDefaults.standard.didShowInformationEditingMessage {
```

Comment:
@rajparad I would like to only show this if other messages aren't triggered above, to prevent competing messages. Can you add an additional conditional here to only show the `editArticleWarning` if lines 221, 223, 225, and 233 were not hit (i.e. no blocked message, protected page message, other errors, or edit notices)? Just keeping track of a simple flag will suffice. Thanks!

Reply:
sure :). I will update and let you know